Public/Invoke-DockerPush.ps1

```diff
@@ -4,7 +4,6 @@ function Invoke-DockerPush {
     [CmdletBinding()]
     param (
         [Parameter(ValueFromPipelineByPropertyName = $true)]
-        [ValidateNotNull()]
```

Comment:
Could you add a test case that tests what happens when you pass in a null value? 

As I understand it, Add-Postfix will throw an exception in that scenario.

Reply:
Add-Postfix checks if it's null or white space:
https://github.com/3shapeAS/dockerbuild-pwsh/blob/master/Private/Utilities.ps1#L45L47

So i think we're safe. Will add more tests like you suggested.

Reply:
Added test in DockerBuild. `$null` will not break anything because it is overriden by the default value.

Reply:
Added `$null` value tests for it.